content/en/event-streaming.md

```diff
@@ -0,0 +1,31 @@
+---
+title: Event Streaming
+status: Feedback Appreciated
+category: concept
+---
+
+## What it is
+
+Event streaming is an approach where software sends event data from one application to another to continuously communicate what they are doing.
+Picture a service broadcasting everything it does to all other services.
+Each activity taken by a service is referred to as an event, hence event streaming.
+For example, the NASDAQ gets updates on stock and commodities pricing every second. 
+An application that monitors a specific set of stocks would need to receive that information in near real-time. 
+If you had an application that monitored a specific set of stocks, you would want to receive that information in near real-time. 
+If NASDAQ provides an API that send the information from their application to yours, you have event streaming.
```

Comment:
```suggestion
NASDAQ provides an [API](https://glossary.cncf.io/application-programming-interface/) that sends (or streams) the information (or events) from their application to any application that subscribes to it.
```

NASDAQ does provide an API, so I think we can say so

Reply:
> 

I could use something else such as AlphaVantage. I wanted to avoid naming a specific tech product as well as highlight someone who is a member with CNCF.  I will change to Yahoo!